jetty-servlets/src/main/java/org/eclipse/jetty/servlets/DoSFilter.java

```diff
@@ -381,7 +381,7 @@ protected void doFilter(HttpServletRequest request, HttpServletResponse response
                 // or if we were woken up we insist or we fail.
                 Boolean throttled = (Boolean)request.getAttribute(__THROTTLED);
                 long throttleMs = getThrottleMs();
-                if (throttled != Boolean.TRUE && throttleMs > 0)
+                if (Boolean.FALSE.equals(throttled) && throttleMs > 0)
```

Comment:
is `FALSE.equals(x)`  the same as `!TRUE.equals(x)` when null values are considered?